CLI/dsi2lsl.c

```diff
@@ -56,6 +56,7 @@ int CheckError( void ){
  * This value is set to 2 seconds, which is sufficient for most operations
  */
 #define BUFFER_SECONDS 2
+#define CHUNK_SIZE 9
```

Comment:
Are there any benefits or drawbacks to increasing or decreasing this? Add some documentation to help others understand why this is needed.

Reply:
We are checking with others to make sure all the sample rates we support (300, 600, etc) have the same chunk size. We'll wait to hear back before merging.